experimental/segmenter/src/complex.rs

```diff
@@ -55,7 +55,11 @@ impl LstmPayloads {
     ) -> Result<Option<DataPayload<LstmDataV1Marker>>, DataError> {
         match provider.load(DataRequest {
             locale: &DataLocale::from(locale),
-            metadata: Default::default(),
+            metadata: {
+                let mut m = DataRequestMetadata::default();
+                m.silent = true;
```

Comment:
Thought: I worry a little bit that this change will hide real errors, because it is actually a bug if `km` data isn't present, because then any `km` text won't be segmented properly. The errors that had been printed before were not very useful or understandable, but that doesn't mean they weren't problems to be fixed.

Reply:
I thought this was in scope and that segmenter would work properly without them. So revert here?